src/lightning_lite/plugins/__init__.py

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from lightning_lite.plugins.environments.cluster_environment import ClusterEnvironment
-from lightning_lite.plugins.io.checkpoint_plugin import CheckpointIO
-from lightning_lite.plugins.io.torch_plugin import TorchCheckpointIO
-from lightning_lite.plugins.io.xla_plugin import XLACheckpointIO
+from lightning_lite.plugins.io.checkpoint_io import CheckpointIO
+from lightning_lite.plugins.io.torch_ import TorchCheckpointIO
```

Comment:
Maybe torch_io, xla_io to avoid the weird torch_ ?

Reply:
@awaelchli, I see you thumbs-upped this comment, but underscore was your suggestion: https://github.com/Lightning-AI/lightning/pull/15058#discussion_r991509209

I don't mind either option. This can be changed later anyways

Reply:
I think in this case it's a good suggestion. Either way is fine with me.

Reply:
Doing this in a different PR

Reply:
I don't like this tbh this is again a inconsistency. If we really want to get rid of the weird `torch_`, I'd rather rename it to `pytorch`, but not add a inconsistency again just for this case (which is even weirder IMO)

Reply:
This disagreement is why I chose to do this in a separate PR. Please :+1: or :-1: #15189 and I'll close or proceed based on the results.

Reply:
yes, just wanted to present an alternative :D